### PR TITLE
Refactor jarvis4

### PR DESCRIPTION
**Pull request by djabarovgeorge**
Make a refactor to the jarvis project inspired by @LetItRock

### CODE REVIEW

File: scripts/jarvis.js [hunk 1]
```diff
@@ -163,3 +161,77 @@ Everything is running 🎊
...
+    function onExit() {
+      command.kill('SIGINT');
+    }
+
+    process.on('SIGTERM', onExit);
+    process.on('SIGINT', onExit);
+
+    command.on('exit', (exitCode) => {
+      if (parseInt(exitCode) !== 0) {
+        reject(new Error(exitCode));
```
Comment (scopsy) [inline scripts/jarvis.js:213]:
```suggestion
        return reject(new Error(exitCode));
```

File: scripts/jarvis.js [hunk 2]
```diff
@@ -163,3 +161,77 @@ Everything is running 🎊
     return true;
   });
 }
+
+const informAboutInitialSetup = () => {
+  const rlp = require('readline');
```
Comment (scopsy) [inline scripts/jarvis.js:166]: Is that a Node API? because if it's an npm package it will crash because the user hasn't run the npm install project yet.

**Review by scopsy — APPROVED** [review]

File: scripts/jarvis.js (hunk 2 above)
Comment (djabarovgeorge) [reply scripts/jarvis.js:166]: yes it is https://www.geeksforgeeks.org/node-js-readline-module/

Comment (scopsy) [reply scripts/jarvis.js:166]: Oh nice didn't know that !

**Pull request merged**